src/presenters/pop-overs/project-options-pop.js

```
@@ -24,7 +24,22 @@ const ProjectOptionsContent = ({ addToCollectionPopover, ...props }) => {
     props.togglePopover();
   }
 
+  function isTeamProject() {
+    const { currentUser, project } = props;
+    for (const team of currentUser.teams) {
```

stupid q but teams is always defined and at least an empty [ ] right?

---

it _should_ be -- currentUser still uses the old API that has everything on it.
if this turns out to not be the case i'd rather this crash than fail silently.